src/data/payments/PaymentHelper.ts

```diff
@@ -75,6 +75,8 @@ export default class PaymentHelper extends Helper<PaymentData, Payment> {
 
   /**
    * Returns the URL the customer should visit to make the payment. This is to where you should redirect the consumer.
+   *
+   * @deprecated Use `payment.getCheckoutUrl()` instead.
    */
   public getPaymentUrl(): Nullable<string> {
     return this.links.checkout?.href ?? null;
```

Comment:
Since the method is so simple it wouldn't bring you much, but you could even do:
```suggestion
  public getPaymentUrl(): Nullable<string> {
    return this.getCheckoutUrl();
```